mapcomposer/app/static/externals/gxp/src/script/plugins/WFSSearchBox.js

```diff
@@ -124,8 +129,13 @@ Ext.namespace("gxp.plugins");
     /** api: method[addOutput]
      */
     addOutput: function(config) {
-        
-    	var controls = (!this.markerFadeoutEnable && !this.noButton) ? [ this.removeMarkerBtn, this.combo] : [this.combo];
+    	var controls;
+    	if(this.itemsContainer && (!this.markerFadeoutEnable && !this.noButton) )
+    		controls = [ new Ext.form.CompositeField( { 
+    			//layout:'hbox',
+    			items:[this.removeMarkerBtn, this.combo]})];	
```

Comment:
if you are using 1 removeMarkerBtn for all the search combo, move it below all the search boxes.